.editorconfig

```diff
@@ -1,18 +1,16 @@
 ; This file is for unifying the coding style for different editors and IDEs.
 ; More information at http://editorconfig.org
 
-root = true
+root = false
```

Comment:
Shouldnt this stay true? as this is the root config for this repo.

Reply:
Pulled this from elsewhere...